.github/workflows/main.yml

```diff
@@ -3,18 +3,32 @@ on:
     types: [labeled]
 
 jobs:
-  trivial:
+  base:
     runs-on: ubuntu-latest
     if: contains(github.event.pull_request.labels.*.name, 'ci:run')
-    name: First Pass Trigger Testing
+    name: Main Tests
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v2
-      - name: Show Workspace Context
-        run: echo 'Workspace is ${{ github.workspace }}'
-      - name: Nasty Hack To Get Right Docker Build Context
-        run: cp ./.github/actions/docker_action/Dockerfile . && cp ./.github/actions/docker_action/action.yml .
-      - name: Run Docker Tests
-        uses: ./
-      - name: Show commit
-        run: git show -q
+      - name: test_all.sh running from Dockerfile.micro
+        uses: docker://jpwithers/tflite-micro-tests
+#       uncomment below to run only the five minute test bit.
+#        with:
+#          args: /bin/sh -c "/opt/tflm/tensorflow/lite/micro/tools/ci_build/test_all.sh"
```

Comment:
is comment still valid? I would expect that directly running test_all.sh would be the same as using Dockerfile.micro.

Reply:
It is valid. Those lines override the CMD in the container which removes the GITHUB_PRESUBMIT argument, thus cutting out the bazel stuff.

Reply:
ah, yes.

I'm going to be soon enabling many more tests from test_all.sh so its time will also increase.

If you need something quick for your work, I might suggest changing this to be  /opt/tflm/tensorflow/lite/micro/tools/ci_build/test_x86.sh (with a comment that says, uncomment to run only the x86 tests for faster development of the CI work)